SyncLounge documentation

MagicalCodeMonkey opened the pull request:
Figuring out some of this was very dependent on [this comment](https://github.com/samcm/synclounge/issues/125#issuecomment-578373503) as well as monitoring the development console for issues.

See https://github.com/GhostWriters/DockSTARTer/pull/875 for the corresponding app PR.

MagicalCodeMonkey committed:
Initial commit for SyncLounge documentation

MagicalCodeMonkey committed:
Linting and minor changes

MagicalCodeMonkey committed:
Linting

nemchik reviewed (changes requested):
I think it would make sense to submit a PR to https://github.com/linuxserver/reverse-proxy-confs with a proxy for synclounge. I also think this one might need a bit of work. Specifically the proxy example here seems to be a subdomain proxy but is using a base url for some things. To simplify things it would make more sense to separate a subfolder and subdomain proxy (or skip the subfolder proxy if it's problematic).

MagicalCodeMonkey commented:
PR in LSIO reverse-proxy-confs: https://github.com/linuxserver/reverse-proxy-confs/pull/136

I will update this documentation soon to reflect that

MagicalCodeMonkey committed:
Update the documentation
- Reflect new proxy confs that are now in LSIO
- Change some wording and information

MagicalCodeMonkey commented:
PRs in SyncLounge to:
- Resolve some issues with the webapp. including the webroot issues: https://github.com/samcm/synclounge/pull/133
- Make some updates to how settings are handled to follow with ENV best practices: https://github.com/samcm/synclounge/pull/132

MagicalCodeMonkey committed:
Merge branch 'master' into synclounge

MagicalCodeMonkey committed:
Update SyncLounge documentation

MagicalCodeMonkey committed:
Linting

MagicalCodeMonkey committed:
Merge branch 'master' into synclounge

MagicalCodeMonkey committed:
Update documentation

MagicalCodeMonkey committed:
Fix Advanced section steps
